js/anims-lasers.js

```diff
@@ -0,0 +1,7 @@
+function shootLaser($city) {
```

Comment:
does this really need its own file? a lot of HTTP overhead for a small bit of code

Reply:
fixed in 1c35b68